agency/coordinate.hpp

```diff
@@ -796,8 +796,8 @@ namespace std
 {
 
 
-template<size_t I, class Tuple>
-struct tuple_element;
+//template<size_t I, class Tuple>
+//struct tuple_element;
```

Comment:
If this declaration is causing problems, it should be completely eliminated, not just commented out.